users/signals.py

```diff
@@ -18,19 +18,23 @@
 #
 # Signal for other modules to activate user
```

Comment:
What does activating or deactivating mean in practice? There is no concept of active or inactive users. When is this emitted?

Reply:
This is from the time when I used is_active from the user.

And the concept is still there (default django functionality). is_active user cannot login for example.

https://github.com/TampereHacklab/mulysa/pull/103/files/7a6263f1a4085fd484b0bb618c884ff87947e1d4#diff-98185baf554572b17e7545c526949658R200 emits those activate_user and deactivate_user signals when users is_active attribute changes.